moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -917,4 +921,32 @@ void JogCalcs::jointJogCB(const control_msgs::JointJogConstPtr& msg)
     latest_command_stamp_ = msg->header.stamp;
 }
 
+bool JogCalcs::changeDriftDimensions(moveit_msgs::ChangeDriftDimensions::Request& req,
+                                     moveit_msgs::ChangeDriftDimensions::Response& res)
+{
+  shared_variables_.drift_dimensions[0] = req.drift_x_translation;
+  shared_variables_.drift_dimensions[1] = req.drift_y_translation;
+  shared_variables_.drift_dimensions[2] = req.drift_z_translation;
+  shared_variables_.drift_dimensions[3] = req.drift_x_rotation;
+  shared_variables_.drift_dimensions[4] = req.drift_y_rotation;
+  shared_variables_.drift_dimensions[5] = req.drift_z_rotation;
```

Comment:
I tried making these member variables of JogCalcs and it fails a test where it is expecting a warning.  I haven't figured out why so for now they are staying in the shared_variables.